### PR TITLE
Add L7 Load Balancer

### PR DESCRIPTION
**Pull request by davi17g**

### CODE REVIEW

File: aerospike-proximus/examples/gke/as-proximus-gke-values.yaml [hunk 1]
```diff
@@ -25,9 +25,11 @@ proximusConfig:
     ticker-interval: 10
 
 service:
-  enabled: true
+  enabled: false
+  tyoe: LoadBalancer
```
Comment (arrowplum) [inline aerospike-proximus/examples/gke/as-proximus-gke-values.yaml:29]: @davi17g did this work? It looks like a typo

Comment (davi17g) [reply aerospike-proximus/examples/gke/as-proximus-gke-values.yaml:29]: @arrowplum It is because LB is on istio side, it's should be like that

Comment (arrowplum) [reply aerospike-proximus/examples/gke/as-proximus-gke-values.yaml:29]: "tyoe"?

**Comment by arrowplum** [comment]
I think this needs to be merged just to have a helm chart which supports 0.4.0 (regardless of demo app)

**Review by arrowplum — APPROVED** [review]
@davi17g I have tested this and run through multiple times. All good for me. Please merge when you feel ready.

File: docs/index.yaml [hunk 2]
```diff
@@ -143,10 +125,28 @@ entries:
...
+    - Aerospike
+    - Vector
+    maintainers:
+    - email: developers@aerospike.com
+      name: Aerospike
+    name: aerospike-vector-search
+    type: application
+    urls:
+    - https://aerospike.github.io/helm-charts/aerospike-vector-search-0.1.0.tgz
+    version: 0.1.0
```
Comment (arrowplum) [inline docs/index.yaml:145]: Shouldn't this version be 0.2.0?

**Review by arrowplum — APPROVED** [review]

File: docs/index.yaml (hunk 2 above)
Comment (davi17g) [reply docs/index.yaml:145]: The tag in `aerospike-vector-search-examples` repo is `0.1.0` that's why decided to leave it as it is basically overwriting the existing one

Comment (arrowplum) [reply docs/index.yaml:145]: Let us go up a version so we do an upgrade. I think we don't need to rely on that tag.

**Comment by arrowplum** [comment]
Merging

File: docs/index.yaml (hunk 2 above)
Comment (davi17g) [reply docs/index.yaml:145]: @arrowplum No problem will rename the tag on dockerhub to be `0.2.0`

**Pull request merged**